xarray/tests/test_cftime_offsets.py

```diff
@@ -1313,7 +1313,7 @@ def test_calendar_year_length(
     assert len(result) == expected_number_of_days
 
 
-@pytest.mark.parametrize("freq", ["Y", "M", "D"])
+@pytest.mark.parametrize("freq", ["Y", "ME", "D"])
 def test_dayofweek_after_cftime_range(freq: str) -> None:
     result = cftime_range("2000-02-01", periods=3, freq=freq).dayofweek
     expected = pd.date_range("2000-02-01", periods=3, freq=freq).dayofweek
```

Comment:
You could probably get away with using `xr.date_range` here, which would now handle translating the frequency automatically, though it wouldn't be quite as clear that it is using `pd.date_range` under the hood:
```suggestion
    expected = xr.date_range("2000-02-01", periods=3, freq=freq).dayofweek
```